#request
uint8 pinNumber
bool value #False for LOW, True for HIGH
---
string message
